File: uri/ad-hock/frecuencia_de_los_numeros/1171.cpp
#include <iostream>

using namespace std;

int main() {
    int n;
    cin >> n;
    
    int frecuencia[2001] = {0};
    
    for (int i = 0; i < n; i++) {
        int x;
        cin >> x;
        frecuencia[x]++;
    }
    
    for (int i = 1; i <= 2000; i++) {
        if (frecuencia[i] > 0) {
            cout << i << " aparece " << frecuencia[i] << " vez(es)" << endl;
        }
    }
    
    return 0;
}

File: uri/ad-hock/frecuencia_de_los_numeros/1171.rs
use std::io;

fn main() {
    let mut input = String::new();
    io::stdin().read_line(&mut input).unwrap();
    let n: usize = input.trim().parse().unwrap();
    
    let mut frecuencia = [0; 2001];
    
    for _ in 0..n {
        input.clear();
        io::stdin().read_line(&mut input).unwrap();
        let x: usize = input.trim().parse().unwrap();
        frecuencia[x] += 1;
    }
    
    for i in 1..=2000 {
        if frecuencia[i] > 0 {
            println!("{} aparece {} vez(es)", i, frecuencia[i]);
        }
    }
}
